app/assets/javascripts/discourse/app/app.js

```diff
@@ -1,10 +1,8 @@
-/* eslint-disable simple-import-sort/imports */
```

Comment:
if it's not needed - does this rule still work?

Reply:
Yeah I was surprised about this, but removing these comments is a no-op even before this eslint upgrade. The only change is that eslint now errors-out for unnecessary eslint-disable comments.

It seems like simple-import-sort ignores the sorting of these imports, because we have another block of imports a few lines below (after the registerDiscourseImplicitInjections() call). 

Not sure if that's a bug or a feature of simple-import-sort... but it works in our favour for now 😅